src/Compilers/Server/VBCSCompilerTests/VBCSCompilerTests.csproj

```diff
@@ -33,8 +33,8 @@
     <ProjectReference Include="..\..\CSharp\Portable\CSharpCodeAnalysis.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Xml" />
+    <Reference Include="System" Condition="'$(TargetFramework)' != 'netcoreapp2.0'" />
+    <Reference Include="System.Xml" Condition="'$(TargetFramework)' != 'netcoreapp2.0'" />
```

Comment:
Why did we need this? Would not expect that from moving forward as the SDK should be compatible. Did we check with the SDK team if this was a known breaking change?

Reply:
I checked with Wes for `System.Configuration` and I assume `System` and `System.Xml` are the same. I'll double-check with him for those too.

The situation is that assemblies with dangling type-forwards are included in `Microsoft.NETCore.App` package (dangling means that the nuget transitive set will not provide the assembly that is forwarded to). Unless we're pulling in a library that has a dependency on those, we should not reference such assemblies, but rather directly reference the package that has the types we need.

Reply:
That said, I'm not sure why the SDK change cause this to surface. I'll let them know.